service/s3/s3manager/batch_test.go

```diff
@@ -439,8 +438,8 @@ func TestBatchDeleteList_EmptyListObjects(t *testing.T) {
 	// called on it
 	iter := &DeleteListIterator{
 		Bucket: input.Bucket,
-		Paginator: request.Pagination{
-			NewRequest: func() (*request.Request, error) {
+		Paginator: aws.Pager{
+			NewRequest: func() (*aws.Request, error) {
```

Comment:
These tests can be updated to use the ListObjectsRequest(input).Paginate() returned pager value.